tools/system_libs.py

```diff
@@ -1873,6 +1874,15 @@ def install_system_headers(stamp):
   cmake_dest = shared.Cache.get_sysroot_dir('lib', 'cmake')
   copytree_exist_ok(cmake_src, cmake_dest)
 
+  # Create a version header based on the emscripten-version.txt
+  version_file = os.path.join(shared.Cache.get_include_dir(), 'emscripten/version.h')
```

Comment:
Is there a particular reason for generating this file, rather than just checking it in?

Reply:
The main reason is that it would mean separately maintaining `emscripten-version.txt` and `version.h` which is annoying and risks them becoming out of sync.

Secondly, these header as never used as checked-in, they are first installed in the system.. so the choice is between copying and file and creating one.  There is no in-place usage anymore.  Just FYI.